docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+services:
+  web:
+    restart: always
+    build:
+      context: ./server
+      dockerfile: Dockerfile
+    volumes:
+      - ./server:/app
+    ports:
+      - "8000:8000"  # Django server
+      - "8765:8765"  # WebSocket server
+    env_file:
+      - .env
+    depends_on:
+      - db
+    networks:
+      - app_network
+
+  db:
+    image: postgres:13
+    restart: always
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_DB: "kundestyrt_db"
+      POSTGRES_USER: "admin"
+      POSTGRES_PASSWORD: "admin"
+    networks:
+      - app_network
+
+  client:
+    restart: always
+    build:
+      context: .
+      dockerfile: client/Dockerfile
+    ports:
+      - "5173:5173"
+    networks:
+      - app_network
+    command: npm run dev -- --host
```

Comment:
do we need this command both in this file and the client Dockerfile?

Reply:
No, we don't. Good catch. I'll delete it from this file.